src/component/nuclear/card/MoviePromoCard.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import  star  from 'src/assets/star.svg';
+
+const MoviePromoCard = ({ movieData }) => {
+
+  const {title, description, genres, ratings, release_date, image_url, director, age_certification, duration} = movieData;
```

Comment:
create a interface for the data type , rather than specifying field during initialisation